functions/index.js

```diff
@@ -4,6 +4,37 @@ const cors = require('cors')({origin: true});
 const admin = require('firebase-admin');
 admin.initializeApp();
 
+exports.saveAudioSuggestions = functions.https.onRequest(async (req, res) => {
+  const uuidv1 = require('uuid/v1');
+  console.log('Saving audio to GCS.');
+  var bucket = admin.storage().bucket('barnard-project-audio');
+  const options = {
+    metadata: {
+      contentType: 'audio/mpeg',
+    }
+  };
+  var uuid = uuidv1();
+  var file = bucket.file(`suggestions/${uuid}.mp3`);
```

Comment:
I thought this cloud function was going to be transcoding from webm (Chrome's native audio recording format) into mp3. We need to figure out how the client and backend interact here. Have you tried calling this from the client?

Reply:
Yes I was using POSTMAN to send binary files to debug. Is that what you mean?